src/components/panel/panel.js

```diff
@@ -35,12 +35,12 @@ angular
  *   var panelRef;
  *
  *   function showPanel($event) {
- *     var panelPosition = $mdPanelPosition
```

Comment:
Unless you changed something, these cannot be directly injected as they as objects, not services. So they must be initialized with new. And because they require dependency injection, the helper methods were created that inject the necessary pieces and return the new object.

I think the previous version was more correct then.